root/static/scripts/relationship-editor/generic.js

```diff
@@ -142,13 +142,6 @@ import {SERIES_ORDERING_TYPE_AUTOMATIC} from '../common/constants';
 
             pushInput(prefix, "target", relationship.target(vm.source).gid);
 
-            _.each(['entity0_credit', 'entity1_credit'], function (prop) {
-                var value = clean(relationship[prop]());
-                if (value) {
```

Comment:
This is also wrong, since `''` is falsey, so no field would get outputted if you cleared the credit.

But even before this, an edit would still get submitted, even though no field was outputted. How? Well, see the lines I'm changing at https://github.com/metabrainz/musicbrainz-server/pull/107/files#diff-9dca8abb5df5f6c32c4c0ecb6a3c4ea4R194